recipes/hypothesis/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set version = "3.4.2" %}
+{% set md5 = "b180f47bdbebdbbbfe4a6d2d7f95c640" %}
+{% set pkgname = "hypothesis" %}
+
+package:
+  name: {{ pkgname }}
+  version: {{ version }}
+
+source:
+  fn: simplejson-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ pkgname[0] }}/{{ pkgname }}/{{ pkgname }}-{{ version }}.tar.gz
+  md5: {{ md5 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - simplejson
+  run:
+    - python
+    - django
+    - enum34  # [py<34]
+
+test:
+  imports:
+    - hypothesis
+    - hypothesis.extra
+    - hypothesis.extra.django
```

Comment:
Drop this line.

Since this package doesn't depend on Django, this import will cause an ImportError, when this module attempts to import `djano`